client/src/app/components/page/list/list.component.ts

```diff
@@ -80,4 +87,13 @@ export class ListComponent implements OnInit, OnDestroy {
             placeholder: 'Choose a tag'
         }
     }
+    public getDateString(date:Date){
```

Comment:
You can use angular pipe, so you can delete this function. Anyway for date / time / timezone manipulation, `moment` library is much more convinient. You can check it here https://momentjs.com/

Reply:
Deleted